### DISCUSSION TITLE
get error message with php8.1 in departament-availability.php

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1860] [General]
https://github.com/LiveHelperChat/livehelperchat/blob/d31b3d96e969255dc2c095d7452f1bee9817a3f5/lhc_web/modules/lhcron/departament-availability.php#L17

get following error message with php8.1
`PHP Deprecated:  DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated in /var/www/html/lhc001/lhc_web/modules/lhcron/departament-availability.php on line 17`

to fix the line should be:
`$date_utc = new \DateTime("", new \DateTimeZone("UTC")); `

please check.

**Comment by remdex** [comment]
Fixed in this commit - https://github.com/LiveHelperChat/livehelperchat/commit/df4f47f6f204146bba8ba4ebc8f0f7ea8a061b8b
